ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa_hcp-4.17-candidate-x86-loaded-upgrade-from-4.16.yaml

```diff
@@ -0,0 +1,67 @@
+base_images:
+  cli-ocm:
+    name: cli-ocm
+    namespace: ci
+    tag: latest
+  rosa-aws-cli:
+    name: rosa-aws-cli
+    namespace: ci
+    tag: latest
+build_root:
+  image_stream_tag:
+    name: ci-tools-build-root
+    namespace: ci
+    tag: latest
+images:
+- dockerfile_path: prow/Dockerfile
+  to: ocp-qe-perfscale-ci-hcp-loaded-upgrade-416to417
+promotion:
+  to:
+  - namespace: ci
+    tag: latest
+releases:
+  latest:
+    release:
+      channel: candidate
+      version: "4.17"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: loaded-upgrade417-24nodes
+  cron: 0 16 10,24 * *
+  steps:
+    cluster_profile: aws-perfscale-qe
+    env:
+      AVAILABLE_UPGRADE: "yes"
+      CHANNEL_GROUP: candidate
+      COMPUTE_MACHINE_TYPE: m5.2xlarge
+      DISABLE_WORKLOAD_MONITORING: "false"
+      ENABLE_AUTOSCALING: "false"
+      GC: "false"
+      HOSTED_CP: "true"
+      ITERATION_MULTIPLIER_ENV: "2"
+      LONG_CLUSTER_NAME_ENABLED: "true"
+      MULTI_AZ: "true"
+      NODE_UPGRADE_TIMEOUT: "14400"
+      OCM_LOGIN_ENV: staging
+      OPENSHIFT_VERSION: 4.16.13
```

Comment:
is there a way for us to set not this specific version when creating the cluster, rather just 4.16? I have seen this sometimes in past jobs that the verison specified is so out of date

Reply:
In this case, it's always picking up 4.16.14 which does not have an upgrade path :( 
To by-pass, I had to hard code.